.env

```
@@ -18,7 +18,7 @@ NETWORK_NUM_ROUNDS=30000
 INDEXER_URL="https://github.com/algorand/indexer"
 NODE_ARCHIVAL="False"
 INDEXER_BRANCH="tzaffi/box-search"
-INDEXER_SHA=""
+INDEXER_SHA="7e7f849dd51ca41d0a8293bcf72f478726a3e036"
```

nice workaround!

---

NOTE: Revert once indexer PR is in good shape and settled down
```suggestion
INDEXER_SHA=""
```